src/main/resources/templates/webshop/index.html

```diff
@@ -209,6 +209,7 @@ <h5 class="modal-title" id="shoppingBasketTitle">
             <div class="modal-footer">
                 <form id="orderForm" method="POST" th:action="@{'/checkout'}" th:object="${orderProduct}">
                     <div id="products"></div>
+                    <input type="checkbox" name="agreedGTC"> I agree to the <a href="https://ch.tudelft.nl/wp-content/uploads/Deelnemersvoorwaarden_versie_12_06_2023.pdf">General Terms and conditions</a></input></br>
```

Comment:
Pls make this a variable like the one in the email

Reply:
I now put it as a variable in the yaml.